targets/mbedos5/tools/cmsis.h

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
This file only got a body, not a license header, as requested. Please add a proper Apache 2.0 license header.